remoulade/state/middleware.py

```diff
@@ -34,8 +36,17 @@ def save(self, message, state_name, **kwargs):
             # Keyword arguments exceed maximum size to
             #  display do not save them
             kwargs_state = {}
+        priority = remoulade.get_broker().get_actor(actor_name).priority
```

Comment:
broker is given as an argument of `after_enqueue`, ... use it instead of this global function.